package.json

```diff
@@ -6,6 +6,7 @@
   "contributors": [
     "Yuri Astrakhan <yurik@wikimedia.org>"
   ],
+  "files": ["index.js"],
```

Comment:
Want to add README.md too?  "README" is pulled in by default but I don't think that the .md files will be unless listed here.

Reply:
npm publish now includes.
```
npm notice 725B  LICENSE     
npm notice 8.0kB index.js    
npm notice 559B  package.json
npm notice 666B  CHANGELOG.md
npm notice 3.2kB README.md 
```
:-)

Reply:
( seems to be clever enough )